docs_source_files/content/webdriver/driver_requirements.en.md

```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
 This was after Google announced the ability to run Chrome headlessly, 
 something also now offered by Mozilla's Firefox.
 
+### Chocolatey
 
+On Windows systems the driver binaries can be installed individually, or as a collection using [chocolatey](https://chocolatey.org/).
```

Comment:
The PR should also include the package managers for `macOS` and `Linux`

Reply:
I'd like to add this, but I don't have access to mac or linux systems to check it on.

Reply:
To move forward with this, maybe you can leave the placeholders for the sections.